content/en/learn/governance/module-psm.md

```diff
@@ -17,29 +17,29 @@ Scope: Each PSM interacts with a single underlying vault type.
+The PSM contract was primarily created to help keep the DAI peg close to $1 during times where DAI demand outstrips DAI supply.
 
 Initially, low liquidation ratio stablecoin vaults were used for this purpose, but this solution proved difficult for Governance to administer due to the risk of stability fees pushing vaults below the 100% collateralization ratio.
 
-The PSM contract allows Governance to collect fees on stablecoins at the point of swap rather than over time.
+The PSM contract allows Governance to collect fees on stablecoins at the point of swap, rather than over time.
 
 ## Trade-offs
 
-A PSM offers the same danger as low liquidation ratio stablecoin vault types: the Maker Protocol will take on a large amount of stablecoin collateral at times where DAI demand outstrips DAI supply. This is usually cited to be a risk due to the centralization of other stablecoins and the possibility of regulatory action targeting Maker specifically. The risk of regulatory action may be slightly greater with a PSM than the alternative because the stablecoin collateral created through a PSM is effectively owned by the Maker Protocol, rather than being owned by a user using it as collateral for a loan of DAI.
+A PSM creates the same danger as low liquidation ratio stablecoin vault types: the Maker Protocol will take on a large amount of stablecoin collateral at times where DAI demand outstrips DAI supply. This is usually cited to be a risk due to the centralization of other stablecoins and the possibility of regulatory action targeting Maker specifically. The risk of regulatory action may be slightly greater with a PSM than the alternative because the stablecoin collateral created through a PSM is effectively owned by the Maker Protocol, rather than being owned by a user using it as collateral for a loan of DAI.
 
 Additionally, the Maker Protocol bears the risk of the underlying collateral stablecoin losing its peg (though this is no different from regular vault-types with stablecoin collateral.)
 
 On the other hand, a PSM offers several advantages:
 
 - Increased DAI stability due to the instant arbitrage opportunity when the DAI price diverges from the price of the collateral type underlying the PSM.
-- Fees are taken upfront on each swap to and from DAI. Because a PSM has no slippage, it is expected that it will attract respectable trading volume if given reasonable fee parameters.
-- There is no requirement to micromanage parameters to ensure continued functioning like there is with low liquidation ratio stablecoin vaults.
+- Fees are taken upfront on each swap to and from DAI. Because a PSM has no slippage, it's able to attract respectable trading volume.
+- There is no requirement to micromanage parameters (via governance actions) to ensure continued functioning, when compared to low liquidation ratio stablecoin vaults.
```

Comment:
"in contrast to [low liquidation...]" would probably work better here, I think

Reply:
Changed!